### PR TITLE
[ADD] migrate l10n_nl

### PR DESCRIPTION
**Pull request by hbrunn**
All the other localizations will need the same thing, I guess?

### CODE REVIEW

File: addons/l10n_nl/migrations/8.0.1.5/noupdate_data.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<openerp>
+    <data>
+        <record id="l10nnl_chart_template" model="account.chart.template">
```
Comment (pedrobaeza) [inline addons/l10n_nl/migrations/8.0.1.5/noupdate_data.xml:3]: l10n_nl_chart_template?

**Comment by pedrobaeza** [comment]
Are chart templates noupdate? At least, l10n_es isn't.

File: addons/l10n_nl/migrations/8.0.1.5/noupdate_data.xml (hunk 1 above)
Comment (hbrunn) [inline addons/l10n_nl/migrations/8.0.1.5/noupdate_data.xml:3]: no: https://github.com/OCA/OCB/blob/8.0/addons/l10n_nl/account_chart_netherlands.xml#L4096

**Comment by hbrunn** [comment]
in 9.0, this one isn't either, but this is about 8.0. Thinking about that: Does that mean we need to lift the noupdate flag when migrating to 9.0? Probably, right?

File: addons/l10n_nl/migrations/8.0.1.5/noupdate_data.xml (hunk 1 above)
Comment (pedrobaeza) [inline addons/l10n_nl/migrations/8.0.1.5/noupdate_data.xml:3]: OK

**Comment by pedrobaeza** [comment]
Yes, that would be desirable to leave a correct state after migration.

**Comment by StefanRijnhart** [comment]
So it should be in the analysis.

**Comment by StefanRijnhart** [comment]
Or in the compare script.

**Comment by hbrunn** [comment]
@StefanRijnhart 'should' as in 'I expect that to happen already' or 'should' as in 'someone should put it in there'?

**Comment by StefanRijnhart** [comment]
Someone, sometime!

**Pull request merged**